api/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@marxan-cloud/api",
-  "version": "0.4.0",
+  "version": "2.0.35",
```

Comment:
now this is a bump

Reply:
more like... nobody ever cared to bump our own version as part of our minimalistic "release" "process" so far 😬 ... we've been on v1.2.x for a while during the previous project phase for a while, then we had a stray v1.3.0.5 and then I lost my patience and took the BDFL decision to just call this phase v2.x 😬